fe/fe-core/src/main/java/com/starrocks/common/TraceManager.java

```diff
@@ -34,31 +35,46 @@ public class TraceManager {
     private static volatile Tracer instance = null;
 
     public static Tracer getTracer() {
-        if (instance == null) {
-            synchronized (TraceManager.class) {
-                if (instance == null) {
-                    if (!Config.jaeger_grpc_endpoint.isEmpty()) {
-                        OpenTelemetrySdkBuilder builder = OpenTelemetrySdk.builder();
-                        SpanProcessor processor = BatchSpanProcessor.builder(
-                                JaegerGrpcSpanExporter.builder().setEndpoint(Config.jaeger_grpc_endpoint)
-                                        .build()).build();
-                        Resource resource = Resource.builder().put("service.name", SERVICE_NAME).build();
-                        SdkTracerProvider sdkTracerProvider = SdkTracerProvider.builder()
-                                .addSpanProcessor(processor)
-                                .setResource(resource)
-                                .build();
-                        builder.setTracerProvider(sdkTracerProvider);
-                        OpenTelemetry openTelemetry = builder.buildAndRegisterGlobal();
-                        instance = openTelemetry.getTracer(SERVICE_NAME);
-                    } else {
-                        instance = GlobalOpenTelemetry.get().getTracer(SERVICE_NAME);
```

Comment:
I didn't find any usages of the global OpenTelemetry instance and I don't expect any use cases within FE components where a Tracer instance is required which cannot be obtained through the TraceManager::getTracer interface. For this reason, the new code constructs a shared Tracer instance and adds span exporters as requested by the config.

Reply:
[Enhancement] Use noop tracer if endpoint not configured (#7724)  Use noop tracer if jaeger endpoint not configured
If nothing is configed(the most common case), this will be used. If I remember correct, this is more efficient. Not an expert here, but Id like to not change things causing uneccessary instability. unless you are absolutely certern.

Reply:
Gotcha. Thank you @decster 
I've pushed an update to this PR to explicitly use the NoOp tracer when tracing endpoints are not configured.